buildpipe/pipeline.py

```diff
@@ -62,12 +62,15 @@ def get_deploy_branch(config: box.Box) -> str:
     return config.deploy.branch or 'master'
 
 
-def get_changed_files(branch: str, deploy_branch: str) -> Set[str]:
+def get_changed_files(branch: str, deploy_branch: str, last_commit_only: bool) -> Set[str]:
     commit = os.getenv('BUILDKITE_COMMIT') or branch
     if branch == deploy_branch:
         command = ['git', 'log', '-m', '-1', '--name-only', '--pretty=format:', commit]
     else:
-        command = ['git', 'whatchanged', '--name-only', '--pretty=format:', 'origin..HEAD']
+        if last_commit_only:
+            command = ['git', 'log', '-1', '--name-only', '--no-merges', '--pretty=format:', 'origin..HEAD']
+        else:
+            command = ['git', 'log', '--name-only', '--no-merges', '--pretty=format:', 'origin..HEAD']
```

Comment:
Thanks for this. Been meaning to change from `whatchanged`.

Reply:
:)